### DISCUSSION TITLE
Documentation for the fields that are filled automatically by using the source parameter?

### DISCUSSION THREAD

**Discussion by jeongukjae** [tensorflow/model-card-toolkit#293] [Ideas]
Hi, there.

How about adding the documentation for the fields that are filled automatically by using [the source parameter](https://github.com/tensorflow/model-card-toolkit/blob/45d3d4ca4ac49f67a8dc34d7cad942f26adcc8df/model_card_toolkit/core.py#L100) or mlmd_source parameter?
For example, I think it should be better if there are docs describing that the following fields are automatically filled when passing the mlmd source ([code](https://github.com/tensorflow/model-card-toolkit/blob/45d3d4ca4ac49f67a8dc34d7cad942f26adcc8df/model_card_toolkit/utils/tf_utils.py#L328-L366)):

* model_details.name
* model_details.version.name
* model_details.references

**Comment by codesue** [comment]
This would be very helpful! It would be nice to have this documentation and a general guide explaining the fields and how to fill out a model card. A first step could be updating that docstring.
